test/e2e/page-objects/flows/snap-permission.flow.ts

```diff
@@ -51,8 +51,8 @@ export async function switchAndApproveDialogSwitchToTestSnap(driver: Driver) {
 export async function completeSnapInstallSwitchToTestSnap(driver: Driver) {
   const snapInstall = new SnapInstall(driver);
   await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-  await snapInstall.clickNextButton();
-  await snapInstall.clickNextButton();
-  await snapInstall.clickNextButton();
```

Comment:
the selectors and names were quite confusing, this has been addressed on the page objects, so now we know exactly which action are we taking